emission/tests/netTests/TestStats.py

```diff
@@ -6,10 +6,88 @@
 
 # Our imports
 from emission.core.get_database import get_client_stats_db, get_server_stats_db, get_result_stats_db
+from emission.net.int_service.giles import archiver
 from emission.net.api import stats
 
 logging.basicConfig(level=logging.DEBUG)
 
+class TestArchiver(unittest.TestCase):
+  def setUp(self):
+    self.path = '/archiver_test_path'
+    self.archiver = archiver.StatArchiver(self.path)
+
+  def tearDown(self):
+    self.archiver.remove()
+
+  # @TODO: Rewrite this test, it doesn't really test much
```

Comment:
I mainly wrote that as a temporary mental note. I would add tests that it works with non-integer timestamps.